nsxt/provider.go

```diff
@@ -145,6 +146,12 @@ func Provider() terraform.ResourceProvider {
 				Description: "Enforcement Point for NSXT Policy",
 				DefaultFunc: schema.EnvDefaultFunc("NSXT_POLICY_ENFORCEMENT_POINT", "default"),
 			},
+			"policy_global_manager": {
```

Comment:
I'd suggest to call it just "global_manager" - Madhu, what do you think?

Reply:
I agree, just "global_manager" is good.